emission/net/api/cfc_webapp.py

```diff
@@ -354,49 +342,19 @@ def deleteUserCustomLabel():
   return { 'label' : to_return }
 
 @post('/result/metrics/<time_type>')
-def summarize_metrics(time_type):
-    logging.debug("Metrics call, finished querying values through skipping")
-    abort(503, "Metrics calls have been temporarily disabled to avoid overloading the server")
-    _fill_aggregate_backward_compat(request)
+def getMetrics(time_type):
+    logging.debug("getMetrics with time_type %s and request %s" %
+                  (time_type, request.json))
+    if time_type != 'yyyy_mm_dd':
+        abort(404, "Metrics calls only supported for yyyy_mm_dd time type")
```

Comment:
In case there is anybody who has still not updated (e.g. have the old app installed, but have turned off auto-updates), I think we should tell them to update so that things keep working. Maybe we can return a different error message?

```suggestion
        abort(404, "Please upgrade to continue using the app dashboard")
```